src/ui/molecules/bridge.burn/bridge.burn.amount.js

```diff
@@ -25,18 +25,27 @@ export const BridgeBurnInput = ({
   });
 
   const formattedAmount = () => {
+    let bigNumberAmount = "0";
+    let formattedAmount = amount == "" ? "0" : amount;
+
     switch (token) {
       case "USDC":
-        return formatter.format(amount);
+        return formatter.format(formattedAmount);
       case "ETH":
-        return formatter.format(amount * ethers.utils.formatUnits(eth_usd, 6));
+        bigNumberAmount = ethers.utils.parseEther(formattedAmount);
+        return formatter.format(
+          ethers.utils.formatUnits(bigNumberAmount.mul(eth_usd), 24)
```

Comment:
More magic numbers here, would be good to move this to a helper function and have documentation as to what's going on.  This also allows for unit testing to be written to ensure we're getting expected values back